src/bindings/python/src/openvino/frontend/jax/jaxpr_decoder.py

```diff
@@ -0,0 +1,283 @@
+            decoder = self.convert_literal_to_constant_node(
+                literal=self.literals[idx], 
+                name=self.name + "/" + f"const({id(node)})", 
+                output_id=id(node)
+            )
+            self.m_decoders.append(decoder)
+            node_visitor(decoder)
+        # Visit every `JaxEqn` in the jaxpr, see https://github.com/google/jax/blob/jaxlib-v0.4.29/jax/_src/core.py#L285
+        for node in self.jaxpr.eqns:
+            literal_decoders = []
+            for inp in node.invars:
+                if isinstance(inp, jax.core.Literal):
+                    literal_decoder = self.convert_literal_to_constant_node(inp)
+                    literal_decoders.append(literal_decoder)
+                    node_visitor(literal_decoder)
+            decoder = JaxprPythonDecoder(node, name=self.name + "/" + node.primitive.name, literals=literal_decoders)
+            self.m_decoders.append(decoder)
+            node_visitor(decoder)
+            
+    def get_op_type(self) -> str:
+        if isinstance(self.jaxpr, jax.core.JaxprEqn):
+            return self.jaxpr.primitive.name
+        else:
+            return "root"
+    
+    def outputs(self) -> List[int]:
+        return [id(v) for v in self.jaxpr.outvars]
+    
+    def output(self, idx: int) -> int:
+        return id(self.jaxpr.outvars[idx])
+    
+    def num_inputs(self) -> int:
+        return len(self.jaxpr.invars)
+    
+    def num_outputs(self) -> int:
+        return len(self.jaxpr.outvars)
+    
+    def as_constant(self):
+        if self.get_op_type() == 'constant':
+            value = self.literals
+            # TODO: dig out how to share the memory.
+            # Currently, using shared_memory will raise `ValueError: array is not writeable``
+            ov_const = jax_array_to_ov_const(value, shared_memory=False)
+            return ov_const.outputs()
+        else:
+            raise ValueError("This is not a constant node so it cannot be converted to a constant.")
+        
+    @staticmethod
+    def convert_param_to_constant_node(jaxpr, param):
+        assert hasattr(jaxpr, 'params'), "The jaxpr does not have params."
+        constant = ivalue_to_constant(jaxpr.params[param], shared_memory=False)
+        return _JaxprPythonConstantDecoder(constant=constant)
+    
+    @staticmethod
+    def convert_literal_to_constant_node(literal, name=None, output_id=None):
+        if isinstance(literal, jax.core.Literal):
+            constant = ivalue_to_constant(literal.val, shared_memory=False)
+        elif isinstance(literal, (jax.Array, np.ndarray)):
+            constant = ivalue_to_constant(literal, shared_memory=False)
+        else:
+            raise TypeError( f"The input should be a literal or jax array, but got {type(literal)}.")
+        return _JaxprPythonConstantDecoder(constant=constant, name=name, output_id=output_id)
+        
+class _JaxprPythonConstantDecoder (Decoder):
```

Comment:
Do we really need a separate decoder for constant? We have `decoder.as_constant()` which can work as constant decoder

Reply:
Agree with Maxim, looks not needed.
@AsakusaRinne, please fix it in separate PR.